### DISCUSSION TITLE
请问各位大神，如何隐藏某个文件夹。

### DISCUSSION THREAD

**Discussion by starscen** [AlistGo/alist#2523] [Q&A]
比如挂载了某个网盘的根目录，但是想要隐藏掉该目录下的某个文件夹（比如“我的相册”这个文件夹），该怎么设置？先行谢过！！

**Comment by zch11230** [comment]
元信息里面可以设置隐藏，不过只是隐藏，手动输入路径也可以访问。

**Reply by fzlychee2019** [reply]
2.3版本是您说的这样，但是3.0版本在无信息里设置后，手动输入也无法访问了。这个非常不方便，还是2.3版本时那样方便。

**Comment by xsd116** [comment]
^网盘某文件夹名称$
^第二个网盘某文件夹名称$
……
